js/apps/system/_admin/aardvark/APP/frontend/js/views/documentsView.js

```diff
@@ -1217,7 +1217,7 @@
 
       if (window.App.naviView && $('#subNavigationBar .breadcrumb').html() !== undefined) {
         $('#subNavigationBar .breadcrumb').html(
-          'Collection: ' + arangoHelper.escapeHtml(this.collectionName.length > 64 ? this.collectionName.substr(0, 64) + "..." : this.collectionName)
+          'Collection: ' + _.escape(this.collectionName.length > 64 ? this.collectionName.substr(0, 64) + "..." : this.collectionName)
```

Comment:
@ajurdak @hkernbach FYI We would probably want to handle the truncation via CSS to avoid information loss here. CSS truncation wasn't widely supported when this code was initially written I think.

Reply:
I would be all for doing the truncation via CSS, but I don't know how to achieve this. Would need someone else to help me out with this.

Reply:
I think it's fine to do this as proposed for now and switch to CSS truncation later. I don't know off the top of my head right now either but maybe @ajurdak or @hkernbach could look into this, possibly in a separate PR.

Reply:
Yes, I wouldn't do this in this PR. Separate PR is fine.